runtime/doc/lua.txt

```diff
@@ -2280,56 +2281,78 @@ find({names}, {opts})                                          *vim.fs.find()*
     searches are recursive and may search through many directories! If {stop}
     is non-nil, then the search stops when the directory given in {stop} is
     reached. The search terminates when {limit} (default 1) matches are found.
-    The search can be narrowed to find only files or or only directories by
+    The search can be narrowed to find only files or only directories by
     specifying {type} to be "file" or "directory", respectively.
+    
+     Examples: >
+     
+       vim.fs.find("aFileOrDirectoryNextToNeovim")
+       => "C:/Users/jdoe/aFileOrDirectoryNextToNeovim"
```

Comment:
I don't think I understand what "next to Neovim" is supposed to mean in these examples. By default, `vim.fs.find()` will find any file with the given name in any child directory, not just the current directory. For examples, just use short, simple file names.

Also, all of these examples assume that the given file actually exists on the filesystem. That assumption should be explicitly stated.

How about something like this instead:

```
For example, assume that Nvim is started in the user's home directory
which contains a file "foo.txt". The home directory also contains a
directory called "bar", and in that directory is a file called "baz.txt".
Then, the following function calls would return the following results:

    vim.fs.find('foo.txt')
    => { '/home/user/foo.txt' }

    vim.fs.find('baz.txt')
    => { '/home/user/bar/baz.txt' }
    
    vim.fs.find('foo.txt', { path = 'bar' })
    => {}

    vim.fs.find('foo.txt', { path = 'bar', upward = true })
    => { '/home/user/foo.txt' }
```